libraries/Microsoft.Bot.Expressions/BuiltInFunctions.cs

```diff
@@ -1517,10 +1527,15 @@ private static Expression RewriteAccessor(Expression expression, string localVar
                 else
                 {
                     var str = expression.ToString();
-                    var prefix = "$global";
+                    var prefix = GlobalVariablePrefix;
                     if (str == localVarName || str.StartsWith(localVarName + "."))
                     {
-                        prefix = "$local";
+                        prefix = LocalVariablePrefix;
+                    }
+                    else if (str == LocalVariablePrefix)
```

Comment:
As i go over the changes, besides the naming changes, here is only place that affect the behavior, right?

Reply:
Right